stock_backend/StoreTickerData.py

```diff
@@ -1,188 +1,141 @@
-StockD =  Stock.reset_index(drop=True).to_dict(orient='index')
-
-StockD = {str(k):v for (k,v) in StockD.items()}
-StockV = StockD.values()
-i = 0
-random_pos = range(len(StockV))
-curr_len = len(random_pos)
-
-IndustryList = Industry.values.tolist()
-SectorList = Sector.values.tolist()
-
-
-IndD ={}
-SectorD = {}
-curr_IndL = {}
-curr_SectorL = {}
-
-#These loops get the number of stocks in each industry and sector and
-# make lists to use to randomly assign a position.
-for ind in IndustryList:
-    stocks = Stock[Stock.industry == ind]
-    if stocks.empty is not True:
-        IndD[ind] = range(len(stocks.index))
-        curr_IndL[ind] = len(IndD[ind])
-
-for sect in SectorList:
-    stocks = Stock[Stock.Sector == sect]
-    if stocks.empty is not True:
-        SectorD[sect] = range(len(stocks.index))
-        curr_SectorL[sect] = len(SectorD[sect])
-
-StocksD = {
-    "NumOfStocks": curr_len,
-}
-
-IndD.pop(np.nan,None)
-SectorD.pop(np.nan,None)
-curr_IndL.pop(np.nan, None)
-curr_SectorL.pop(np.nan,None)
-
-IndDocument = {
-    "Industry":curr_IndL
-}
-SectorDocument ={
-    "Sector":curr_SectorL
-}
-MarketCapDocument = {
-    "MarketCap":MarketCapD
-}
-db.collection('Ticker-Info').document('Industry').set(IndDocument)
-db.collection('Ticker-Info').document('Sector').set(SectorDocument)
-db.collection('Ticker-Info').document('Stock').set(StocksD)
-db.collection('Ticker-Info').document('Market-Cap').set(MarketCapDocument)
-
-for Info in StockV:
-    NewD = {
-        'Symbol': Info['Symbol'],
-        'Name': Info['Name'], 
-        'MarketCap': Info['MarketCap'],
-        'IPOyear': Info['IPOyear'],
-        'Sector': Info['Sector'],
-        'Industry': Info['industry'], 
-        'MarketCapSize': MarketCapSymbolD[Info['Symbol']],
-        # Used to increase ease of querying
-        'RandomPos' :random_pos[randrange(curr_len)],
-        "MarketCapPos": -1 if MarketCapSymbolD[Info['Symbol']] is "N/A" else MarketCapDictL[MarketCapSymbolD[Info['Symbol']]][randrange(MarketCapD[MarketCapSymbolD[Info['Symbol']]])],
-        'IndustryPos': -1 if Info['industry'] is np.nan or Stock[Stock.industry == Info['industry']].empty else IndD[Info['industry']][randrange(curr_IndL[Info['industry']])],
-        'SectorPos' : -1 if Info['Sector'] is np.nan or Stock[Stock.Sector == Info['Sector']].empty else SectorD[Info['Sector']][randrange(curr_SectorL[Info['Sector']])]
-    }
-    db.collection('Ticker-Info').document('Stock').collection('Stocks').document(Info['Symbol']).set(NewD)
-    i+=1
-    curr_len-=1
-    if Info['industry'] is not np.nan:
-        curr_IndL[Info['industry']]-=1 
-    if Info['Sector'] is not np.nan:
-        curr_SectorL[Info['Sector']]-=1
-    if MarketCapSymbolD[Info['Symbol']] is not "N/A":
-        MarketCapD[MarketCapSymbolD[Info['Symbol']]]-=1
-    print(str(i)+ "\\" + str(len(StockV)) + ":" + Info['Symbol'] +" has been added to the database with IPO year " +  str(Info['IPOyear']))
-
+import pandas as pd
```

Comment:
This file shouldn't be changed at all.  And the older version has more updated code. Don't use this one.

Reply:
@jack-y-xu can you explain why you changed, or if you just were using an old version?

Reply:
was using old version that worked, new version wouldn't work

Reply:
Same as below